crates/pathfinder/examples/calculate_commitments.rs

```diff
@@ -55,7 +55,10 @@ fn main() -> anyhow::Result<()> {
 
         let now = Instant::now();
         let (transaction_commitment, event_commitment) = (
-            calculate_transaction_commitment(&transactions)?,
+            calculate_transaction_commitment(
+                &transactions,
+                pathfinder_lib::state::block_hash::FinalHashType::Normal,
```

Comment:
This is not yet entirely correct -- we have to get the version number for these blocks as well...

I'll fix this.